storm-core/src/clj/backtype/storm/cluster.clj

```diff
@@ -489,10 +489,10 @@
         (let [path (backpressure-path storm-id node port)
               existed (exists-node? cluster-state path false)]
           (if existed
-            (if (not on?)
-              (delete-node cluster-state path))   ;; delete the znode since the worker is not congested
-            (if on?
-              (set-ephemeral-node cluster-state path nil acls))))) ;; create the znode since worker is congested
+            (when (not on?)
```

Comment:
It is mostly that I put in some debugging and didn't want an ```(if (do (log-message) ...)))``` to fix an issue, and ended up not changing it back after removing the debug logging.  If you want me to change it back and add back in the comments I can.